recipes/geospatial/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "geospatial" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/geospatial-{{ version }}.tar.gz
+  sha256: 93edf988a6606e21f8b13251c2e8c6c073ef5b8f7b96d51ec38ef8b0a00e4587
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
```

Comment:
The list of reqs is much longer upstream. Are you sure that only the ones below are needed?

Ref: https://github.com/giswqs/geospatial/blob/v0.1.0/requirements.txt

Reply:
This list of reqs has changed since this PR was initially submitted. I have updated the list in the recipe.